writeups/Drupal_Drupal_TameeshB.md

```diff
@@ -0,0 +1,83 @@
+# Contributing to Drupal
+
+## What is Drupal 
+Drupal is a content management software.  
+It's used to make many of the websites and applications you use every day.  
+Drupal has great standard features, like easy content authoring, reliable performance, and excellent security.  
+But what sets it apart is its flexibility; modularity is one of its core principles.  
+[More about the Drupal project here.](https://www.drupal.org/about)
+
+## The Drupal community
+The Drupal community is one of the largest open source communities in the world. 
+We're more than 1,000,000 passionate developers, designers, trainers, strategists, coordinators, editors, and sponsors working together.
+We build Drupal, provide support, create documentation, share networking opportunities, and more. 
+Our shared commitment to the open source spirit pushes the Drupal project forward. 
+New members are always welcome.
+
+## Setting it up
+### Prerequisites  
+* Git
+* XAMPP/LAMP/MAMP (respectively for Windows/Linux/Mac OS)
+* phpmyadmin (XAMPP and MAMP stack already have it installed)
+* Composer
+
+### Installing the server
+* [Video on installing MAMP on Mac OS.](https://drupalize.me/videos/installing-mamp-web-server)
+* [Guide on installing LAMP on Linux.](https://www.digitalocean.com/community/tutorials/how-to-install-linux-apache-mysql-php-lamp-stack-on-ubuntu-16-04)
+* [Guide on installing XAMPP on Windows.](https://www.drupal.org/node/2559943)
+* [Guide on installing phpmyadmin on LAMP stack.](https://www.digitalocean.com/community/tutorials/how-to-install-and-secure-phpmyadmin-on-ubuntu-12-04)
+
+### Setup Steps:  
+1. Download and install XAMPP/LAMP/MAMP depending on the operating system you use.   
+  
+2. Start Apache and MySQL servers.     
+   
+3. Start your terminal and navigate to your web server's directory. Enter one of the following commands:     
+###### on Windows ######  
```

Comment:
Here, I had something like this in mind:

![screenshot from 2017-09-10 12 45 06](https://user-images.githubusercontent.com/16105680/30246964-ecf4faba-9625-11e7-9d35-6fcbacdaa70b.png)